### DISCUSSION TITLE
Proper credit for making Rich Presence and Leaderboards

### DISCUSSION THREAD

**Discussion by julkasalsa** [RetroAchievements/RAWeb#684] [Ideas]
Seperate section on the game page to credit developers who made leaderboards, and rich presence. Those are important parts of the site, and right now there is no clear information who was working on them.

**Comment by televandalist** [comment]
Leaderboards have been taken care of. Now it's just RP.

**Comment by meleu** [comment]
probably something @Tsearo could take a look at (as it's possibly a change that is still fresh in his mind) :pleading_face: :point_right: :point_left:

**Comment by Jamiras** [comment]
Leaderboard and rich presence changes are now tracked through audit comments. Only achievement authors show up in the set authors list.
